packages/amplify-util-uibuilder/src/commands/utils/storageManager.ts

```diff
@@ -0,0 +1,18 @@
+import { StudioForm, StudioSchema } from '@aws-amplify/codegen-ui';
+
+export const hasStorageField = (form: StudioForm): boolean => {
+  const result = Object.values(form.fields).some((field) => {
+    return 'inputType' in field && field.inputType?.type === 'StorageField';
+  });
+
+  return result;
```

Comment:
nit: can we use optional chaining on the `inputType` key instead of checking with `in`?

```suggestion
  return Object.values(form.fields).some((field) => field?.inputType?.type === 'StorageField');
```

Reply:
cleaned up the helper functions utilizing guard functions and `includes`